src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" >
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <div class="card__content">
+
+        <div class="card__item-picture">
+          <img class="card__imac-image"
+          src="./images/imac.jpeg"
+          alt="imac">
+        </div>
+
+        <p class="card__item-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__item-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__item-rating">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <section class="card__reviews">
+            Reviews: 5
+          </section>
+        </div>
+
+        <div class="card__item-price">
+          <section class="card__price-text">
+            Price:
+          </section>
+          <section class="caprice-number">
+            $2,199
+          </section>
+
+        </div>
+        <div class="card__buy-button">
+          <a href="#"
```

Comment:
![image](https://user-images.githubusercontent.com/54936044/223883071-caedfd08-c293-4c6e-a41a-e991a91d7b81.png)

Reply:
Also row 53 and 55, you used two of the same classes, please fix